python/kserve/kserve/model.py

```diff
@@ -246,6 +245,10 @@ def load(self) -> bool:
         self.ready = True
         return self.ready
 
+    def unload(self):
```

Comment:
`unload` is used by model repository to remove a model from repo, maybe we can call it `stop` or `shutdown` here?

Reply:
Cool, I'll change it to `stop`.